gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -302,6 +323,8 @@ image::gateway-quickstart-k8-api-config-endpoint.png["Screenshot of 'Endpoint' p
 https://jsonplaceholder.typicode.com/
 ----
 .. Type `80` in the *Port* field.
++
+The example in this guide assumes an external IP that uses port `80` instead of the default port, `8081`. 
```

Comment:
from seba: maybe create a separate page on all the typical things you can do when deploying Flex, such as

* node affinity
* changing port
* setting cpu and memory (prob need recommendations with x APIs and y "traffic")
* readiness (after Flex starts when Flex is ready to receive traffic)
* HA settings (downgrade and ?)
* other common ones (TBD which and how many)